common/net_sys.h

```diff
@@ -145,6 +145,8 @@ COMPILE_TIME_ASSERT(sockaddr, offsetof(struct sockaddr, sa_family) == SA_FAM_OFF
 
 #ifndef PLATFORM_AMIGAOS3
 #include <sys/param.h>
+#else
+#define __NO_NET_API
```

Comment:
This is universal I guess?

Reply:
EDIT: Or is it related to commenting out of `-I$(OSLIBS)/amigaos/netinclude` down below?

Reply:
Can't really remember why I did that in q_endian.h (was for RoadShow SDK support??), so I don't know what to do with it for now :(

Reply:
That didn't work out, as __amigaos__ is not defined there. I think I'll leave it in net_sys.h as it's a network-related macro.